src/common/components/Auth/ConfirmSignUpNote.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+import { Button } from "@/common/components/Button";
+import { useSearchParams, notFound } from "next/navigation";
+
+export const ConfirmSignUpNote = () => {
+  const searchParams = useSearchParams();
```

Comment:
Hi @davidlhw, good suggestion here, I did not think of using [searchParam](https://nextjs.org/docs/app/api-reference/file-conventions/page#searchparams-optional) as it seems to have some issues for Nextjs v13+ where search params object is empty, but since we are using v14 onwards we can give it a try

I made the changes at [a5030a4](https://github.com/AfterClass-io/afterclass.io-v2/pull/103/commits/a5030a4f553f24243b50eb8ab73bbe8ba2769058) and tested with deployed link, everything works well :))